bundles/org.openhab.binding.carnet/README.md

```diff
@@ -0,0 +1,248 @@
+|              | overallMileage          | Number:Length        | yes     | Overall milage after this trip                                                        |
+| tripLong1    | timestamp               | DateTime             | yes     | Trip time                                                                             |
+|              | avgElectricConsumption  | Number:Energy        | yes     | Electrical consumptio during the trip                                                 |
+|              | avgFuelConsumption      | Number:Volume        | yes     | Average fuel consumption for this trip                                                |
+|              | avgSpeed                | Number:Speed         | yes     | Average Speed for this trip                                                           |
+|              | startMileage            | Number:Length        | yes     | Start milage for the trip                                                             |
+|              | mileage                 | Number:Length        | yes     | Distance for this trip.                                                               |
+|              | overallMileage          | Number:Length        | yes     | Overall milage after this trip                                                        |
+| destination1 | destinationName         | String               | yes     | The textual description of this destination, might be empty/NaN.                      |
+|              | destinationPoi          | String               | yes     | Name of the Point-of-Interested (is this destination has one) or NaN                  |
+|              | destinationLocation     | Location             | yes     | Geo coordinates of this location (Location item type format)                          |
+|              | destinatinStreet        | String               | yes     | Street address of the destination address, might be empty (NaN).                      |
+|              | destinationCity         | String               | yes     | City of the destination address, might be empty (NaN).                                |
+|              | destinationZip          | String               | yes     | The zip code of the destination address, might be empty (NaN).                        |
+|              | destinationCountry      | String               | yes     | Country of the destination address, might be empty (NaN).                             |
+|              | destinationSource       | String               | yes     | Source of the destination, e.g. could be the mobile App, might be empty (NaN).        |
+| doors        | doorFrontLeftLocked     | Switch               | yes     | ON: The left front door is locked.                                                    |
+|              | doorFrontLeftState      | Switch               | yes     | ON: The left front door is open.                                                      |
```

Comment:
I prefer type switch, because the channel handling is generic. Based on the sensor data returned from the API the channels are created dynamically, I have a generic handling, which then ends up it updating the channel with ON/OFF, otherwise I would need to differ between Switch and Contact on a per-channel basis

Reply:
OK